src/main/kotlin/com/github/jengelman/gradle/plugins/shadow/transformers/PropertiesFileTransformer.kt

```diff
@@ -215,6 +226,14 @@ public open class PropertiesFileTransformer @Inject constructor(
   override fun hasTransformedResource(): Boolean = propertiesEntries.isNotEmpty()
 
   override fun modifyOutputStream(os: ZipOutputStream, preserveFileTimestamps: Boolean) {
+    if (conflicts.isNotEmpty()) {
```

Comment:
Based on the review comment above, we can check `Fail` and throw exceptions if `conflicts` is not empty.

Reply:
I misread this, `conflicts` will never be empty if the strategy is not `Fail`. Good to me.